docs/modules/ROOT/examples/hazelcast-enterprise.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast-sample
+spec:
+  clusterSize: 3
+  repository: 'docker.io/hazelcast/hazelcast-enterprise'
+  version: '{full-version}-slim'
```

Comment:
`full-version` contains the `major.minor.patch` version of the operator.

Is this what we want to use?

Reply:
You are right, we don't want to use operator version here. Can we add a compatible Hazelcast version parameter, so we can use it where it is needed?

Reply:
```suggestion
  version: '{page-latest-supported-hazelcast}-slim'
```